documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -525,26 +515,47 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 **Why the rule is important**: Per the ARM guidelines, each swagger specification must contain one resource provider.
 
-**How to fix the violation**: N/A.
+**How to fix the violation**: One swagger specification must have one resource provider. Feel free to create multiple swaggers, each for one provider.
```

Comment:
nit: "Please create multiple ...." instead of "Feel free ....", do stick to this language throughout.